brave/brave-core
# (Wallet) Connect unlock action from WebUI to native unlock screen

## Resolves https://github.com/brave/brave-browser/issues/40224

When Wallet is locked, show a new page that will open native unlock screen when double tapped.


https://github.com/user-attachments/assets/398d81d0-fc32-4d22-b77e-9bb866456dc7








## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Navigate to Wallet section
- Type password to unlock the screen
- Tap on the Wallet icon in the address bar, tap on the overflow menu and select "Lock Wallet"
- Obsererve the WebUI page changes to a locked state asking to double tap
- Double tap on the screen
- Observe the native screen to unlock pops up
- Type the password again
- Observe the Wallet web page is visible
